restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/impl/ExistsFilterOperation.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.service.query.filter.operation.impl;
+
+import io.stargate.db.query.Predicate;
+import io.stargate.web.docsapi.exception.DocumentAPIRequestException;
+import io.stargate.web.docsapi.service.query.filter.operation.CombinedFilterOperation;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+/** Exists filter operation can resolve if any database value exists. */
+@Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
+@Value.Immutable(singleton = true)
+public abstract class ExistsFilterOperation implements CombinedFilterOperation<Boolean> {
```

Comment:
Is this really a filter operation? 

Conceptually it should pass any row that it received on input since the row obviously exists at that point?

Should we perhaps handle "exists" at the query parsing level? We could produce something like a `FilterExpression` with conditions only on the primary key columns (also mentioned in another comment on this PR). WDYT?

Reply:
It is. From the filter operation it's checks that the value is not null. 

Combined with the condition and the rule that each row must have a value, yes each row will pass it. But I don't care much, we need anyway to parse it and place it in the tree, thus I am keeping it as is ;)

Reply:
Actually you are correct here.. This is not a real filter op. Removed!